test/e2e/tests/hardware-wallets/lattice-connect.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { strict as assert } from 'assert';
+import { Suite } from 'mocha';
+import { Driver } from '../../webdriver/driver';
+import FixtureBuilder from '../../fixture-builder';
+import { withFixtures, unlockWallet } from '../../helpers';
+import { isManifestV3 } from '../../../../shared/modules/mv3.utils';
+
+describe('Lattice hardware wallet @no-mmi', function (this: Suite) {
```

Comment:
This testcase is splited from the test file `import-flow.spec.js` because it is more hardware wallet related. Therefore, we should put it in the same folder with other hardware wallet tests. 
This test need to be converted to page object modal in separate PR.